depends/packages/crate_fuchsia_zircon.mk

```diff
@@ -0,0 +1,15 @@
+package=crate_fuchsia_zircon
+$(package)_crate_name=fuchsia-zircon
```

Comment:
My sympathies lie with the poster here: https://users.rust-lang.org/t/rand-crate-depends-on-fuchsia-zircon/15303

>  It seems weird that something as simple and self-contained as a random number generator would depend on language bindings for an experimental, unreleased operating system that I don’t have…

I don't think it makes sense for Zcash's depends system to pull in OS bindings for every weird system under the sun when Zcash is not being built for those systems. ("Weird" is not derogatory here. Fuchsia-Zircon is an awesome capability-based microkernel. I count Windows as weird in this context too.)